configs/codestyles/SquareAndroid.xml

```diff
@@ -257,7 +260,7 @@
     <option name="ALLOW_TRAILING_COMMA" value="true" />
     <indentOptions>
       <option name="INDENT_SIZE" value="2" />
-      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      <option name="CONTINUATION_INDENT_SIZE" value="2" />
```

Comment:
We'll need to reformat the Register Android codebase. I'll take care of this (that was the main concern why #82 was declined). Cash Android should already use a 2 space continuation indent.